### PR TITLE
[VEN-2874] Deploy Oracles to Base Sepolia

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:1]: We need configurations for tokens and their feeds as well. Although these are not needed for deployments but we need them for record keeping. You can refer this PR as well in which we are adding config for opmainnet 
https://github.com/VenusProtocol/oracle/pull/225/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ffR320

File: deployments/basesepolia/BoundValidator_Implementation.json [hunk 1]
```diff
@@ -0,0 +1,651 @@
+{
+  "address": "0x9D6aF23E1E42e1eB341eE81cC91855E1f94aDBA0",
+  "abi": [
```
Comment (GitGuru7) [inline deployments/basesepolia/BoundValidator_Implementation.json:3]: Contracts are not verified

**Review by GitGuru7 — COMMENTED** [review]
We also need to add new network in ci.yml

File: deployments/basesepolia/BoundValidator_Implementation.json (hunk 1 above)
Comment (chechu) [reply deployments/basesepolia/BoundValidator_Implementation.json:3]: Done

File: helpers/deploymentConfig.ts
Comment (chechu) [reply helpers/deploymentConfig.ts:1]: https://github.com/VenusProtocol/oracle/pull/235/commits/173940065940f2c1ee3d5bc0fdf7b903403d8755

**Review by chechu — APPROVED** [review]

**Pull request merged**